frontend/packages/dev-console/integration-tests/features/Helm/actions-on-HelmRelease.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Helm Chart
+    User should able to perform the actions on the helm releases
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+
+@regression, @smoke
+Scenario: Perform Upgrade action on Helm Release through Context Menu: HR-08-TC02
+    Given user right clicks on the Helm Release
```

Comment:
Action should not be performed in Given statement, make sure of it

Reply:
Updated the statement.